dev_scripts/pytest-wrapper.py

```diff
@@ -0,0 +1,55 @@
+#!/usr/bin/env python3
+# -*- coding: utf-8 -*-
+
+# PyTest Wrapper script - temporary solution to tests failing non-deterministically [1]
+# This is only fixed in podman v4.3.0. The wrapper essentially runs the tests in sequence
+# when the podman version is lower.
+#
+# [1]: https://github.com/freedomofpress/dangerzone/issues/217
+
+# FIXME this whole script should be removed and the respective Makefile calling line
+# replaced once all supported platforms have a podman version >= v4.3.0.
+
+import re
+import subprocess
+import sys
+
+import pytest
+from pkg_resources import parse_version
+
+from dangerzone.container import get_runtime_name
+
+PODMAN_MIN_VERSION = "4.3.0"
+
+
+def get_podman_version():
+    result = subprocess.run(
+        "podman version --format '{{.Client.Version}}'", shell=True, capture_output=True
+    )
+    version = result.stdout.decode()[:-1]  # trim trailing \n
+    return version.split("-dev")[0]  # exclude "-dev" suffix from version
```

Comment:
Fun fact, Python 3.9 added [`str.removesuffix()`](https://docs.python.org/3/library/stdtypes.html#str.removesuffix). However, we can't use it yet, because Ubuntu Focal has Python 3.8 :-/. Anw, not actionable, just wanted to share this fact.

Reply:
That's quite useful. Too bad that we can't use it :/